tests/testthat/helper-bs4_book.R

```diff
@@ -0,0 +1,24 @@
+
+create_minimal_bs_book <- function(path,
```

Comment:
The current recommendation is to store helpers in R/ but this works too (I personally prefer it) https://blog.r-hub.io/2020/11/18/testthat-utility-belt/#code-called-in-your-tests